src/server/pkg/collection/collection.go

```diff
@@ -80,7 +80,7 @@ func (c *collection) indexDir(index Index, indexVal string) string {
 	indexDir := c.prefix
 	// remove trailing slash
 	indexDir = strings.TrimRight(indexDir, "/")
-	return fmt.Sprintf("%s__index_%s/%s", indexDir, index, indexVal)
```

Comment:
Interesting. I'll revert it, since it's not relevant to the substance of this PR. I thought we did migrations using the pfs api now, though?